runatlantis.io/docs/using-atlantis.md

```diff
@@ -124,6 +124,49 @@ Because Atlantis under the hood is running `terraform apply plan.tfplan`, any Te
 They're ignored because they can't be specified for an already generated planfile.
 If you would like to specify these flags, do it while running `atlantis plan`.
 
+---
+## atlantis import
+```bash
+atlantis import [options] ADDRESS ID -- [terraform import flags]
+```
+### Explanation
+Runs `terraform import` that matches the directory/project/workspace.
+This command discards the terraform plan result. After an import and before an apply, another `atlantis plan` must be run again.
+
+### Examples
+```bash
+# Runs import
+atlantis import ADDRESS ID
+
+# Runs import in the root directory of the repo with workspace `default`
+atlantis import -d . ADDRESS ID
+
+# Runs import in the `project1` directory of the repo with workspace `default`
+atlantis import -d project1 ADDRESS ID
+
+# Runs import in the root directory of the repo with workspace `staging`
+atlantis import -w staging ADDRESS ID
+```
+
+::: tip
+* If import for_each resources, it requires a single quoted address.
+  * ex. `atlantis import 'aws_instance.example["foo"]' i-1234567890abcdef0`
+:::
+
+### Options
+* `-d directory` Import a resource for this directory, relative to root of repo. Use `.` for root.
+* `-p project` Import a resource for this project. Refers to the name of the project configured in the repo's [`atlantis.yaml`](repo-level-atlantis-yaml.html) repo configuration file. This cannot be used at the same time as `-d` or `-w`.
+* `-w workspace` Import a resource for a specific [Terraform workspace](https://developer.hashicorp.com/terraform/language/state/workspaces). Ignore this if Terraform workspaces are unused.
+
+### Additional Terraform flags
+
+If you need to run `terraform import` with additional arguments, like `-var 'foo=bar'` or `-var-file myfile.tfvars`
+you can append them to the end of the comment after `--`, ex.
+```
+atlantis imoport -d dir 'aws_instance.example["foo"]' i-1234567890abcdef0 -- -var foo='bar'
+```
+If you always need to append a certain flag, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
```

Comment:
Please try to omit the word "you"

```suggestion
If a flag is needed to be always appended, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
```

Reply:
This sentence is based on plan/apply command sencencese. Thanks! I will try to improve docs.